File: hash.cpp
#include <iostream>
#include <unordered_map>

int main() {
    std::unordered_map<std::string, int> ht = {{"apple", 10}, {"banana", 20}, {"cherry", 30}};

    for (const auto& p : ht)
        std::cout << p.first << " -> " << p.second << "\n";

    return 0;
}

File: Самостоятельная работа №2/hash.java
import java.util.HashMap;
import java.util.Map;

public class Main {
    public static void main(String[] args) {
        Map<String, Integer> ht = new HashMap<>();
        ht.put("apple", 10);
        ht.put("banana", 20);
        ht.put("cherry", 30);

        for (var entry : ht.entrySet()) {
            System.out.println(entry.getKey() + " " + entry.getValue());
        }
    }
}

